packages/cli/tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "extends": "@webstudio-is/tsconfig/base.json",
+  "exclude": ["./templates/*"],
```

Comment:
Why not check templates?

Reply:
Example, 
https://github.com/webstudio-is/webstudio/blob/add-typecheck-for-cli/packages/cli/templates/route-template.ts#L114

`ASSET_BASE` is not imported in this file. But the generated template from this can import `ASSET_BASE`. 

https://github.com/webstudio-is/webstudio/blob/add-typecheck-for-cli/packages/cli/src/__generated__/router.ts#L8

And many of the imports that are using relative path doesn't actually exist until the file is injected into users project. So, it's raising un-wanted errors.

Reply:
.ts is also wrong extension. Should be .tsx

Reply:
The extension to doesn't really matter, because we are taking it as a string in build phase. In `prebuild` when routes are actually created from this. They are marked as `tsx`

https://github.com/webstudio-is/webstudio/blob/main/packages/cli/src/prebuild.ts#L349

Reply:
This is the error, if we don't ignore the files under templates
![Screenshot 2023-09-01 at 13 02 04](https://github.com/webstudio-is/webstudio/assets/11075561/2b314371-0fae-4134-9dca-0242a7ee0f81)

The files `../__generated_/index.css` won't exist in the cli. They are available only in the users repo. So, they are no actual ts bugs. SO ignored those files. We do, so the same for `eslint` too.
https://github.com/webstudio-is/webstudio/blob/main/.eslintrc.js#L9

Reply:
Let's still rename to .tsx caz it's hard to maintain utility changes when typescript is trashing module with syntax errors.

Reply:
ok, changed and pushed again 👍